### PR TITLE
Add jlfabi's Suffield file

### LINKED ISSUE #26: jlfabi's Workflow for adding Suffield to the Repository
## Suffield, CT
- [x] Create and issue
- [x] Create a branch
- [x] Make changes
- [x] Open a PR
- [x] Collaborate/Have a discussion
- [ ] Get approval to :ship: 
- [ ] Merge

### PR DESCRIPTION
**Pull request by ghost**
closes #26

### CODE REVIEW

File: jlfabi-suffield.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Suffield, CT
+
+### Restaurants
+- Three Figs
+
+### Places to Go
+- The town green
```
Comment (ghost) [inline jlfabi-suffield.md:7]: Not much going on in Suffield, CT

File: jlfabi-suffield.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Suffield, CT
+
+### Restaurants
+- Three Figs
```
Comment (ghost) [inline jlfabi-suffield.md:4]: Nice place to have drinks

**Review by ghost — COMMENTED** [review]
Review

File: jlfabi-suffield.md (hunk 2 above)
Comment (jaimemendozadev) [reply jlfabi-suffield.md:4]: If I went to Three Figs, what burger would you recommend and what wine pairing should I get?

**Review by jaimemendozadev — APPROVED** [review]

**Pull request merged**